uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -149,7 +149,8 @@
 <xsl:variable name="ABSOLUTE_MEDIA_PATH" select="concat($CONTEXT_PATH,'/',$MEDIA_PATH)"/>
 <xsl:variable name="SKIN_RESOURCES_PATH" select="concat('/',$MEDIA_PATH,'/',$SKIN,'/skin.xml')"/>
 <xsl:variable name="SKIN_PATH" select="concat($ABSOLUTE_MEDIA_PATH,'/',$SKIN)"/>
-<xsl:variable name="PORTAL_SHORTCUT_ICON" select="concat($CONTEXT_PATH,'/favicon.ico')" />
+<xsl:variable name="SHORTCUT_ICON" select="concat($CONTEXT_PATH,'/favicon.ico')" />
```

Comment:
I like your change to `RenderingPipelineConfiguration`.

It doesn't look like the extra `xsl:variable` (`SHORTCUT_ICON`) is needed any longer.

Reply:
The SHORTCUT_ICON is needed for the default value when the property isn't set